src/Symfony/Component/EventDispatcher/EventDispatcher.php

```diff
@@ -94,7 +94,7 @@ public function getListeners(string $eventName = null)
     public function getListenerPriority(string $eventName, $listener)
     {
         if (empty($this->listeners[$eventName])) {
-            return null;
+            return 0;
```

Comment:
This looks wrong to me. The interface states this:

> Returns null if the event or the listener does not exist.

Making this change here violates the contract defined by the interface.